app/models/matchup.rb

```diff
@@ -2,33 +2,76 @@
 #
 # Table name: matchups
 #
-#  id          :bigint           not null, primary key
-#  winner_id   :bigint
-#  position    :integer
-#  bracket_id  :bigint           not null
-#  option_a_id :bigint
-#  option_b_id :bigint
-#  created_at  :datetime         not null
-#  updated_at  :datetime         not null
+#  id             :bigint           not null, primary key
+#  winner_id      :bigint
+#  position       :integer
+#  bracket_id     :bigint           not null
+#  option_a_id    :bigint
+#  option_b_id    :bigint
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  round          :integer          default(1)
+#  round_position :integer          default(1)
 #
 class Matchup < ApplicationRecord
   belongs_to :bracket
   belongs_to :winner, class_name: "Entry", optional: true
   belongs_to :option_a, class_name: "Entry", optional: true
   belongs_to :option_b, class_name: "Entry", optional: true
 
+  after_update :send_winner_to_next_matchup, if: proc {|matchup| matchup.winner.present? }
```

Comment:
Really nice! 👍🏽

It's also possible to [use a method instead](https://guides.rubyonrails.org/active_record_callbacks.html#using-if-and-unless-with-a-symbol) of a proc, if you prefer. That might look something like:

```rb
class Matchup < ApplicationRecord
  ...
  after_update :send_winner_to_next_matchup, if: :winner_is_present?

  def winner_is_present?
    winner.present?
  end
end
```